kpops/component_handlers/__init__.py

```diff
@@ -11,6 +11,20 @@
 
 
 class ComponentHandlers:
+    _instance: ComponentHandlers | None = None
+
+    def __new__(
+        cls,
+        schema_handler,
+        connector_handler,
+        topic_handler,
+        *args,
+        **kwargs,
+    ) -> ComponentHandlers:
+        if not cls._instance:
+            cls._instance = super().__new__(cls, *args, **kwargs)
+        return cls._instance
```

Comment:
Singletone? 🥹

Reply:
yep, still feels a little like a naive singleton, but I consider this a valid solution